### DISCUSSION TITLE
How to start using (before init of a repo)

### DISCUSSION THREAD

**Discussion by lalamax3d** [spoiledcat/git-for-unity#8] [Q&A] [answered]
Hi, i have installed and it is working fine (ubuntu 20, unity 2020.3).
Q. before i init a repo. i wanted to ask. Should i change any setting from unity preferences version control.. or should it stay (visible meta files) or change to something else.

![image](https://user-images.githubusercontent.com/567387/202370113-e711f68a-88a5-4a47-ab59-4f27a7dd7611.png)

**Comment by CapnRat** [comment, marked as answer]
No change is needed. Visible Meta Files is the expected setting. :)

**Reply by lalamax3d** [reply]
thankyou very much
